examples/source/interactivity-dynamic-content/Show_More_Button.html

```diff
@@ -82,7 +82,8 @@
     </amp-list-load-more>
   </amp-list>
 
-  <!-- You can implement a show more button by using `amp-list` and `amp-bind`, where
+  <!-- ## Show more using amp-list and amp-bind -->
+  <!-- You can also implement a show more button manually by using `amp-list` and `amp-bind`, where
   `amp-list` `src` data are bind to the value of an `amp-state` element. Here we are using two `amp-state`: `productsState`
```

Comment:
one nit (didn't see it earlier): `productsState` doesn't really make sense anymore. Use `photos` instead.

Reply:
PTAL